parseRecipes.js

```diff
@@ -9,9 +9,19 @@ var difficulties = {
   'Beginner': 1
 };
 
-var categoryOrder = ['General Usage', 'Offline', 'Beyond Offline', 'Performance', 'Web Push'];
+var categories = module.exports.categories = [
+  { title: 'General Usage', slug: 'general-usage' },
```

Comment:
In the demo but rarely in titles and headers so far. What about adding it to at least one section to see if it helps ranking.